elements/bulbs-video/components/revealed.js

```diff
@@ -156,13 +160,23 @@ export default class Revealed extends React.Component {
     return false;
   }
 
-  vastUrl (videoMeta) {
-    let baseUrl = 'http://us-theonion.videoplaza.tv/proxy/distributor/v2?rt=vast_2.0';
+  getProfValue () {
+    if (window.isMobile.any) {
+      return 'theonion_mobileweb_html5';
+    } else {
+      return 'theonion_desktop_html5';
+    }
+  }
 
+  vastUrl (videoMeta) {
+    let baseUrl = `http://${SETTINGS.FREEWHEEL_NETWORK_ID}v.fwmrm.net/ad/g/1?`;
```

Comment:
@MelizzaP missing a `.` between the network id and the `v`.  Example (`http://1b656.v.fwmrm.net/ad/g/1?...`)